test/404.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Why there's a 404 page in markdown here?

Reply:
Oops